public/index.pug

```diff
@@ -25,24 +25,24 @@ html(lang="en-US")
     link(rel="canonical" href=pageUrl)
 
   body(class="no-js order-alphabetically")
-    header(class="header" role="banner")
+    header(class="header")
       ul(class="header__list")
         li(class="header__list-item")
-          a(href="https://github.com/simple-icons/simple-icons" rel="noopener") GitHub
+          a(href="https://github.com/simple-icons/simple-icons") GitHub
         li(class="header__list-item")
-          a(href="https://www.npmjs.com/package/simple-icons" rel="noopener") NPM
+          a(href="https://www.npmjs.com/package/simple-icons") NPM
         li(class="header__list-item")
-          a(href="https://packagist.org/packages/simple-icons/simple-icons" rel="noopener") Packagist
+          a(href="https://packagist.org/packages/simple-icons/simple-icons") Packagist
         li(class="header__list-item")
-          a(href="https://www.jsdelivr.com/package/npm/simple-icons" rel="noopener") JSDelivr
+          a(href="https://www.jsdelivr.com/package/npm/simple-icons") JSDelivr
         li(class="header__list-item")
-          a(href="https://unpkg.com/browse/simple-icons/" rel="noopener") Unpkg
+          a(href="https://unpkg.com/browse/simple-icons/") Unpkg
       h1(class="header__title") Simple Icons
       p(class="header__description") #{iconCount} Free #[abbr(title="Scalable Vector Graphic") SVG] icons for popular brands
 
     div(class="sticky-header-shadow" aria-hidden)
```

Comment:
While improving the html, it would be good to change this for `aria-hidden="true"`
Right now, it renders `aria-hidden=""` which is an invalid value

Reply:
Thanks for the suggestion, I always assumed `aria-hidden`/`aria-hidden=""` has the same effect as `aria-hidden="true"` but indeed it is not. I'll fix all instances of this across the website :+1: